doc/2/guides/getting-started/8-deploy/index.md

```diff
@@ -4,4 +4,150 @@ type: page
 title: Deploy
 description: Deploy your application on a remote server
 order: 800
----
\ No newline at end of file
+---
+
+# Deploy
+
+Kuzzle is just a Node.js application that also needs Elasticsearch and Redis to run.  
+
+The only specifity is that Kuzzle needs to compile C and C++ dependencies so the `npm install` will also be looking for `python`, `make` and `g++` packages.   
+
+At Kuzzle we like to use Docker and Docker Compose to quickly deploy applications.  
+In this guide we will see how to deploy a Kuzzle application on a remote server.
+
+**Prerequisites:**
+  - A SSH access to a remote server running on Linux
+  - [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on this server
```

Comment:
I think we really need to add "A Kuzzle custom application" as a prerequisite to this guide (not sure about the wording)

Reply:
Since it's still the getting started people are supposed to have a small application by now

Reply:
Yes but if they find this page using the Algolia search it can be confusing to don't know this prerequisite